securedrop_client/gui/widgets.py

```diff
@@ -3918,6 +3930,26 @@ def __init__(self, source: Source, controller: Controller) -> None:
 
         self.addAction(DeleteConversationAction(self.source, self, self.controller))
         self.addAction(DeleteSourceAction(self.source, self, self.controller))
+        self.addAction(DownloadSelectedConversation(self, state, self.controller))
```

Comment:
slight pref for a rename to: `DownloadConversationAction`, similar to `DeleteConversationAction`

Reply:
I went for `DownloadConversation` (no `...Action` suffix) because eventually I'd see the actions defined in `gui/conversation/actions.py` and used with a prefix:

```python
from securedrop_client/gui/conversation import actions

#     prefix --v                          v-- suffix would be redundant in the 'actions' module
self.AddAction(actions.DownloadConversation(self, state, self.controller))
```